### PR TITLE
Add next post link

### PR DESCRIPTION
**Pull request by gokerlek**
Transition links between post page have been created

### CODE REVIEW

File: components/post-navigator.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            color: #4b7bec;
+            font-weight: 300;
+          }
+        `}
+      </style>
+    </div>
+  );
+};
+
+export const getStaticProps = async () => {
```
Comment (onerzafer) [inline components/post-navigator.js:35]: This is not necessary here

File: components/use-next-post.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link';
+
+export const useNextPost = (page, posts) => {
+  const indexNumberPath = posts.page.list;
```
Comment (onerzafer) [inline components/use-next-post.js:4]: Instead d of indexNumberPath simply call it blogPosts

File: components/use-next-post.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  const indexLength = indexNumberPath.length;
```
Comment (onerzafer) [inline components/use-next-post.js:5]: Instead if indexLength just length

File: components/use-next-post.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  const indexNumber = indexNumberPath.findIndex(
+    (index) => index.content.id === page.id,
+  );
+  const nextPageData =
+    indexNumberPath[indexNumber + 1 === indexLength ? indexNumber : indexNumber + 1];
+  const nextPageTitle = nextPageData.content.title;
+  const nextPagePath = nextPageData.path.replace('blog', '');
+  const nextPageId = nextPageData.content.id;
+
+  return (
```
Comment (onerzafer) [inline components/use-next-post.js:15]: Sins this is a hook it should not return jsx but an object. Yiu can add the jsx part at Page navigator

File: components/use-previous-post.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link';
+
+export const usePreviousPost = (page, posts) => {
```
Comment (onerzafer) [inline components/use-previous-post.js:3]: Same comments as useNextPost

File: components/use-previous-post.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+  const previousPageData =
+    indexNumberPath[indexNumber - 1 === -1 ? indexNumber : indexNumber - 1];
+  const previousPagePath = previousPageData.path.replace('blog', '');
+  const previousPageTitle = previousPageData.content.title;
+  const previousPageId = previousPageData.content.id;
+
+  return (
+    <>
+      {indexNumber - 1 === -1 ? null : (
```
Comment (onerzafer) [inline components/use-previous-post.js:17]: Could be simpler if you check indexNumber === 0

**Review by onerzafer — CHANGES_REQUESTED** [review]

File: components/use-next-post.js (hunk 4 above)
Comment (gokerlek) [reply components/use-next-post.js:15]: where is the jsx part

**Review by onerzafer — APPROVED** [review]

**Pull request merged**